docs/reference/volumes/index.md

```diff
@@ -7,10 +7,12 @@ The following volume types are available in {{brand}} for persistent block stora
 If you create a volume without specifying a volume type, then the default volume type applies.
 
 
-| Volume type    | Default          | [Encryption](../../howto/openstack/cinder/encrypted-volumes.md) |
-| -----------    | -------          | ------------                                                    |
-| `cbs-standard` | :material-close: | :material-check:                                                |
-| `cbs-premium`  | :material-check: | :material-check:                                                |
+| Volume type              | Default          | [Encryption](../../howto/openstack/cinder/encrypted-volumes.md) |
+| -----------              | -------          | ------------                                                    |
+| `cbs-standard`           | :material-close: | :material-close:                                                |
+| `cbs-premium`            | :material-check: | :material-close:                                                |
```

Comment:
`cbs-premium` actually _isn't_ the default volume type as defined in the Cinder API (that's `cbs-standard`), but it is indeed being selected by default in Cleura Cloud Management Panel. That happens to be a UI bug in CCMP, thanks for pointing that out! We'll make sure that it gets fixed.